source/UF/H-Levels.lagda

```diff
@@ -1,5 +1,7 @@
 Martin Escardo and Ian Ray, 06/02/2024
```

Comment:
This authorship is kind of misleading. I don't think I had any real contribution! There still is an alternative file `HLevels` which I wrote in 2019, but that file is very short and also assumes univalence, while here you get away with function extensionality only. Moreover, there many more additions here, done all by yourself. May I suggest that you make yourself as the sole author?

(Eventually I may remove the old `HLevels` file and use yours only, when this PR is pushed.)